src/ipc.rs

```diff
@@ -152,7 +153,7 @@ impl<T> IpcSender<T> where T: Serialize {
                 let os_ipc_shared_memory_regions;
                 let os_ipc_channels;
                 {
-                    let mut serializer = bincode::Serializer::new(&mut bytes);
+                    let mut serializer = bincode::Serializer::<_, LittleEndian>::new(&mut bytes);
```

Comment:
Shouldn't that rather be `NativeEndian`?

(Which is apparently hard-aliased to `LittleEndian` anyway, for whatever reason -- but still it seems a more obvious choice conceptually?...)